Create hogpatch-operations.md (#13043)
PostHog/posthog.com: contents/handbook/people/hogpatch-operations.md

> Although Hogpatch is primarily catered to YC founders to cultivate their ideas in our coworking space, we encourage employees at PostHog who live in the Bay Area or are visiting to enjoy the space as well. If you have any questions that aren’t already addressed in this section here, please reach out to **Judy** or **Scott**.

Reviewer: ```suggestion
Hogpatch is our San Francisco coworking space, shared with a handpicked group of our YC users. PostHog teammates in the Bay Area (or visiting) are welcome to enjoy the space too.
```

**<TeamMember name="Judy Opperwall" photo />**, our Office Manager, is on site **Mondays to Thursdays from 9am–5pm**. For any issues while she is not in, please reach out to the `#project-hogpatch` channel or DM her directly.

> 🚧 The space is still a WIP (est. opening **Nov 2025**) so proactively flag things that need to be fixed.

---

> ## Building Access

Reviewer: ```suggestion
## Building access
```

- If you are visiting for the first time, ring the **black intercom doorbell** on the front door.
- <TeamMember name="Judy Opperwall" photo /> will be notified and will remotely unlock the door for you.
- In the future, we will set up an **NFC reader** for guests/visitors that will automatically expire after a set time.

> - These passes will be sent to **YC founders and PostHog visitors via email invite**.

Reviewer: ```suggestion
- These passes will be sent to **YC founders and PostHog visitors via email invite**.
- There’s no check-in or reservations needed, it's a very relaxed setup.

```

---

## Parking

- We have a garage in the building that allows up to **4–5 cars max**.
- This is for **internal employees only**, and spots are limited.
- Please ask **<TeamMember name="Judy Opperwall" photo />** if you’d like a **fob**.

> - If you don’t plan to drive in often, let <TeamMember name="Judy Opperwall" photo /> know beforehand so she can let you in/out on the day you visit.

Reviewer: - Always remember to close the garage door when leaving the premises. If you're worried you have forgotten, <TeamMember name="Judy Opperwall" photo /> can double check this via CCTV.

---

## Housekeeping

### How to make service/supply requests

- If you notice anything missing or that needs replenishing, reach out to <TeamMember name="Judy Opperwall" photo /> on the `#project-hogpatch` channel.

> - We’ll have teething problems while we figure out how folks use the space—so bear with us.

Reviewer: ```suggestion
- We’ll have teething problems while we figure out how folks use the space, so bear with us. We are constantly learning.
```

> - If you have specific requests, ask <TeamMember name="Judy Opperwall" photo />.

Reviewer: similar to line 34 - can remove
